packages/auth/src/api/Login.ts

```diff
@@ -9,51 +9,34 @@
 *
 */
 
-import { AbstractSession, ImperativeExpect, Logger, ImperativeError, RestConstants, SessConstants, HTTP_VERB } from "@zowe/imperative";
+import { AbstractSession, ImperativeExpect, Logger, RestConstants, SessConstants } from "@zowe/imperative";
 import { ZosmfRestClient } from "../../../rest";
+import { LoginConstants } from "./LoginConstants";
 
 /**
- * Class to handle logging onto z/OSMF.
+ * Class to handle logging onto APIML.
  * @export
  * @class Login
  */
 export class Login {
 
     /**
-     * Perform z/OSMF login to obtain LTPA2 or other token types.
+     * Perform APIML login to obtain LTPA2 or other token types.
      * @static
      * @param {AbstractSession} session
      * @returns
      * @memberof Login
      */
-    public static async login(session: AbstractSession, request: HTTP_VERB, resource: string) {
+    public static async apimlLogin(session: AbstractSession) {
         Logger.getAppLogger().trace("Login.login()");
         ImperativeExpect.toNotBeNullOrUndefined(session, "Required session must be defined");
 
-        // TODO(Kelosky): we eventually will hope to use a real z/OSMF authentication endpoint
-        // get client instance and perform a get on /zosmf/info
         const client = new ZosmfRestClient(session);
         await client.request({
-            request,
-            resource
+            request: "POST",
```

Comment:
Just a question:

The login function does not catch an error or check the response status code of the rest request, but the logout function does those things.

I was curious why they are different.

Reply:
I believe this was an oversight. I will add an error here shortly.

Reply:
I added the error and a test to cover it.